javascript.js

```diff
@@ -1,13 +1,54 @@
 
+var growthButton = document.getElementById("miracleGrow");
+var heightInput = document.getElementById("howTall");
+var charInput = document.getElementById("whatLeaf");
 
+// Function to grow tree on keypress
+function enterKey(event) {
+	var key = event.keyCode;
+	if (key === 13) {
+		toGrow();
+	}	
+}
+
+// Function to build tree
+function toGrow() {
```

Comment:
Check the requirements again. This needs to be a function that takes an argument of an object. So you need to call `toGrow` and pass in the tree object. You're defining the tree object inside `toGrow`.